app/goal_routes.py

```diff
@@ -0,0 +1,93 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, abort, jsonify
+from app.routes_helpers import validate_model
+from app.models.goal import Goal
+
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+
+    if not request_body.get("title"):
+        abort(make_response({"details": "Invalid data"}, 400))
+
+    goal = Goal.from_dict(request_body)
+
+    db.session.add(goal)
+    db.session.commit()
+
+    return make_response(goal.to_dict(), 201)
+
+
+@goals_bp.route("", methods=["GET"])
+def get_all_goals():
+    goals = Goal.query.all()
+
+    goals_response = [goal.to_dict()["goal"] for goal in goals]
```

Comment:
Check below at lines 10-15 in the `models/goal.py` file for additional feedback for this suggestion I'm making. If we get rid of the outer dictionary, we don't need to specify a particular key.
```python
    goals_response = [goal.to_dict() for goal in goals]
```